src/components/CustomCollection/CustomCollection.tsx

```diff
@@ -48,13 +46,29 @@ const CustomCollection = ({
   const urlInputValue = useRef<string>('')
   const addCustomLinkModal = useRef<ModalRef>(null)
   const linkInput = useRef<ComboBoxRef>(null)
+  const [isEditing, setEditing] = useState(false)
 
   useEffect(() => {
     if (isAdding && linkInput.current) {
       linkInput.current.focus()
     }
   }, [isAdding])
 
+  useEffect(() => {
+    // If there is no title, prompt user to enter one
+    if (title === '') {
+      setEditing(true)
+    }
+  }, [])
+
+  const handleSubmitEdit = (e: React.FormEvent<HTMLFormElement>) => {
```

Comment:
as a general rule, I would recommend keeping the event handler itself in the same component as the element that triggers it (in this case, the form) and just pass in the submission handler to that component. That way DOM API stuff is kept contained in the component that renders the elements.